lib/index.js

```diff
@@ -50,10 +50,10 @@ const Rooftop = {
       apiToken: opts.apiToken,
       contentType: (name) => {
         return {
-          get: (opts = {}) => {
-            const params = Object.keys(opts).join(',')
+          get: (opts = { params: {} }) => {
+            const params = Object.keys(opts.params).join(',')
```

Comment:
Hmmm so if someone provides options I think this means the empty params will be cancelled out, whether or not the options they provided contain `params`. Is this ok?